crates/matrix-sdk/src/account.rs

```diff
@@ -261,18 +261,26 @@ impl Account {
     /// # async {
     /// # let homeserver = Url::parse("http://localhost:8080")?;
     /// # let client = Client::new(homeserver).await?;
-    /// let profile = client.account().get_profile().await?;
+    /// let profile = client.account().get_current_user_profile().await?;
     /// println!(
     ///     "You are '{:?}' with avatar '{:?}'",
     ///     profile.displayname, profile.avatar_url
     /// );
     /// # anyhow::Ok(()) };
     /// ```
-    pub async fn get_profile(&self) -> Result<get_profile::v3::Response> {
+    pub async fn get_current_user_profile(&self) -> Result<get_profile::v3::Response> {
```

Comment:
I talked to some folks in the SDK team.

We agreed that we'd prefer this to be called: `fetch_user_profile()`.